src/script/common/c_converter.cpp

```diff
@@ -76,10 +76,7 @@ static void set_vector_metatable(lua_State *L)
 
 void push_float_string(lua_State *L, float value)
 {
-	std::stringstream ss;
-	std::string str;
-	ss << value;
-	str = ss.str();
+	auto str = ftos(value);
 	lua_pushstring(L, str.c_str());
```

Comment:
```suggestion
	lua_pushlstring(L, str.data(), str.size());
```

Reply:
Why? It's a float. It cannot contain NULL characters.

Reply:
It would still save Lua from doing a `strlen`, pretty irrelevant in terms of performance though.

Reply:
But it's probably a tiny little bit faster if it gets the size.
(Not that it really matters, this function is just called from `push_noiseparams`.)